src/pages/Beta/Buy/index.tsx

```diff
@@ -11,16 +21,62 @@ export default function BuyV2() {
     bottom: 0;
   `
 
-  const url = `https://buy.moonpay.io?apiKey=${MOONPAY_PK}`
-  return (
-    <Iframe
-      title="Buy"
-      allow="accelerometer; autoplay; camera; gyroscope; payment"
-      width="100%"
-      height="100%"
-      src={url}
-    >
-      <p>Your browser does not support iframes.</p>
-    </Iframe>
-  )
+  const buy = (provider: string) => {
+    setActiveOnRamp(provider)
+  }
+
+  const back = () => {
+    setActiveOnRamp('')
+  }
+
+  let url
+  if (activeOnRamp === 'coinbase')
+    url = `https://pay.coinbase.com/buy/select-asset?appId=${COINBASE_PK}&destinationWallets=%5B%7B%22address%22%3A%22${walletAddr}%22%2C%22blockchains%22%3A%5B%22avalanche-c-chain%22%5D%7D%5D`
+  if (activeOnRamp === 'moonpay') url = `https://buy.moonpay.io?apiKey=${MOONPAY_PK}`
+  if (url) {
+    return (
+      <>
+        <BackRow>
+          <BackButton variant="plain" onClick={back}>
+            {t('buyPage.back', { defaultValue: 'Back' })}
+          </BackButton>
+        </BackRow>
+
+        <Iframe
+          title="Buy"
+          allow="accelerometer; autoplay; camera; gyroscope; payment"
+          width="100%"
+          height="100%"
+          src={url}
+        >
+          <p>{t('buyPage.iframeSupport', { defaultValue: 'Your browser does not support iframes.' })}</p>
+        </Iframe>
+      </>
+    )
+  } else {
+    return (
+      <TopContainer>
+        <Card>
+          <CardTitle>
+            <Title>Coinbase Pay</Title>
+          </CardTitle>
+          <PayButton variant="primary" onClick={() => buy('coinbase')} isDisabled={!account}>
+            <CBIcon src="./images/coinbase_coin_pay_blue.svg" />
```

Comment:
can you please move this image to `src/assets/svg` folder ?

Reply:
done